pkgs/build-support/dotnet/make-nuget-deps/default.nix

```diff
@@ -1,10 +1,10 @@
 { linkFarmFromDrvs, fetchurl }:
 { name, nugetDeps }:
 linkFarmFromDrvs "${name}-nuget-deps" (nugetDeps {
-  fetchNuGet = { pname, version, sha256
+  fetchNuGet = { pname, version, sha256 ? "", hash ? ""
```

Comment:
Now that these are set by default we should probably check if both are passed since they're mutually exclusive.

Reply:
Won't the `fetchurl` check it by itself? Why should we duplicate this logic here if it anyway passes to `fetchurl`?